packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -0,0 +1,410 @@
+            :class="$style['c-cookieBanner-card']">
+            <div
+                id="dialog1"
+                :class="$style['c-cookieBanner-content']"
+                role="dialog"
+                aria-labelledby="cookieConsentLabel"
+                aria-modal="true">
+                <h2
+                    id="cookieConsentLabel"
+                    :class="$style['c-cookieBanner-title']">
+                    <a
+                        ref="cookieBannerHeadingFocus"
+                        name="cookieConsentTitle"
+                        tabindex="0"
+                        data-consent-title>
+                        {{ copy.mainTitle }}
+                    </a>
+                </h2>
+
+                <p :class="$style['c-cookieBanner-text']">
+                    {{ copy.text1 }}
+                </p>
+                <p :class="$style['c-cookieBanner-text']">
+                    {{ copy.text2 }}
+                </p>
+                <p
+                    :class="$style['c-cookieBanner-text']">
+                    {{ copy.text3 }}
+                    <a
+                        :href="copy.linkHref"
+                        :class="$style['c-cookieBanner-link']"
+                        target="_blank">
+                        {{ copy.linkText }}</a>{{ copy.text4 }}
+                </p>
+            </div>
+
+            <div :class="$style['c-cookieBanner-CTA']">
+                <button-component
+                    tabindex="0"
+                    role="button"
+                    data-button-accept
+                    is-full-width
+                    @click.native="acceptActions">
+                    {{ copy.buttonText2 }}
+                </button-component>
+                <button-component
+                    tabindex="0"
+                    role="button"
+                    button-type="ghost"
+                    data-button-nonaccept
+                    is-full-width
+                    @click.native="nonAcceptActions">
+                    {{ copy.buttonText1 }}
+                </button-component>
+            </div>
+        </div>
+        <div v-if="legacyBanner">
+            <div :class="[$style['c-cookieWarning'], { [$style['c-cookieBanner--hidden']]: hideBanner }]">
+                <div :class="$style['c-cookieWarning-inner']">
+                    <p>
+                        {{ copy.legacyBannerText }}
+                        <a
+                            class="c-cookieWarning-link"
+                            :href="copy.linkHref">
+                            {{ copy.legacyBannerLinkText }}
+                        </a>
+                    </p>
+                    <button
+                        class="c-cookieWarning-btn"
+                        data-test-id="cookieBanner-close-button"
+                        aria-label="Close"
+                        @click="hideLegacyBanner" />
+                </div>
+            </div>
+        </div>
```

Comment:
It's probably worth extracting this into a separate component (within `f-cookie-banner`)

Reply:
Done